tests/lib/test_git.py

```diff
@@ -113,6 +113,13 @@ def test_push(self, io_mock):
             "git push origin feature/branch --tags --follow-tags", "/path_to/a_git_repository",
         )
 
+    def test_rebase(self, io_mock):
+        git.rebase("/path_to/a_git_repository", "feature/branch")
+
+        io_mock.execute.assert_called_once_with(
+            "git rebase origin feature/branch --keep_empty", "/path_to/a_git_repository",
+        )
```

Comment:
```suggestion
        io_mock.execute.assert_called_once_with(
            "git rebase feature/branch --keep_empty", "/path_to/a_git_repository",
        )
```

Maybe i'm missing something out but I don't see where this `origin` comes from...

Reply:
Sorry, I have been a little bit too quick, wrong copy/paste ;)